backend/test/controllers/location_controller_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule WeWorkForBeer.LocationControllerTest do
+  use WeWorkForBeer.ConnCase
+
+  alias WeWorkForBeer.Location
+
+  setup do
+    conn = conn() |> put_req_header("accept", "application/json")
+    {:ok, conn: conn}
+  end
+
+  test "GET /locations", %{conn: conn} do
+    location1 = Repo.insert %Location{
+      name: "South Station",
+      address: "745 Atlantic Ave"
+    }
+    location2 = Repo.insert %Location{
+      name: "DUMBO",
+      address: "81 Prospect Street"
+    }
+
+    conn = get conn, location_path(conn, :index)
+    body = json_response(conn, 200)
+
+    assert body["locations"] == [
```

Comment:
There has to be a better way to do these than manually writing the result. I guess once we write some kind of serializer module.